projectforum/projects/tests/test_project_filter.py

```diff
@@ -0,0 +1,186 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+
+import json
+
+from projectforum.projects.forms import *
+from projectforum.projects.models import Project
+import projectforum.projects.project_filters as project_filters
+
+import test_create_projects
+
+
+class ProjectsTest(TestCase):
+    def setUp(self):
+        self.user_model = get_user_model()
```

Comment:
Right now you aren't using self.user in your tests. Will you need the user object in future tests? If not, we probably don't need to create a user at the beginning of every test